subsys/mgmt/updatehub/updatehub.c

```diff
@@ -762,6 +762,18 @@ static void probe_cb(char *metadata, size_t metadata_size)
 	LOG_INF("Probe metadata received");
 }
 
+int updatehub_confirm(void)
+{
+	return boot_write_img_confirmed();
```

Comment:
This file is missing mcuboot.h include

Reply:
This is an intermediate step and I think this happened because I reordered commits. The final solution is on next commit which add `updatehub_storage_mark_partition_as_confirmed` function. Should I add include here to keep tracking?

Reply:
I think it makes sense so the change is complete.

Reply:
> This is an intermediate step and I think this happened because I reordered commits. The final solution is on next commit which add `updatehub_storage_mark_partition_as_confirmed` function. Should I add include here to keep tracking?

Please do.